tutor/src/screens/assignment-review/homework-scores.js

```diff
@@ -170,14 +174,18 @@ const AssignmentHeading = observer(({ ux, heading }) => (
                 {heading.type}
             </HeadingMiddle>
             <HeadingBottom>
-                {heading.dropped &&
-          <CornerTriangle color="blue"
-              data-test-id="dropped-question-indicator"
-              data-question-id={`Q${heading.question_id}`}
-              tooltip={heading.dropped.drop_method == 'zeroed' ?
-                  'Points changed to 0' : 'Full credit given to all students'}
-          />}
-                {ScoresHelper.formatPoints(heading.displayPoints)}
+                {heading.someQuestionsDropped &&
+                <CornerTriangle color="blue"
+                    data-test-id="dropped-question-indicator"
+                    data-question-id={`Q${heading.question_id}`}
+                    tooltip={heading.everyQuestionZeroed ? 'Points changed to 0' :
+                             (heading.everyQuestionFullCredit ? 'Full credit given to all students':
+                             'Question dropped for one or more students')}
+                />}
+                {heading.someQuestionsDropped &&
+                     !heading.everyQuestionZeroed &&
+                     !heading.everyQuestionFullCredit ?
+                <OrangeInfoIcon/> : ScoresHelper.formatPoints(heading.points)}
             </HeadingBottom>
```

Comment:
I'm not a big fan of using conditional logic in the middle of a bunch of JSX.  To me it seems to make it difficult to reason about what's actually rendering.

How about moving most of the logic into methods on the header class, and then making a `IndicatorTriangle` or something that does this:
```
const IndicatorTriangle = observer((({ heading }) => {
  if (heading.hasNoDroppedConditions) {
      return  <span>{heading.formattedPoints}</span>
  }

  if (heading.hasMixedDroppedQuestions) {
    return <OrangeInfoIcon/>
  }
  return (
     <CornerTriangle color="blue"
                     data-test-id="dropped-question-indicator"
                     data-question-id={`Q${heading.question_id}`}
                     tooltip={heading.everyQuestionZeroed ? 'Points changed to 0' :
                              (heading.everyQuestionFullCredit ? 'Full credit given to all students':
                              'Question dropped for one or more students')}
                 />
  )
})
```